src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -311,9 +313,16 @@ export const View = (props: Props) => {
       >
         <TabList
           tabs={tabsData}
-          onSelectionChange={(selectedKey) =>
-            setSelectedTab(selectedKey as TabType)
-          }
+          onSelectionChange={(selectedKey) => {
+            setSelectedTab(selectedKey as TabType);
+            const buttonId =
+              "header_" +
+              ((selectedKey as TabType) == "fixed" ? "fixed" : "action_needed");
```

Comment:
Looks like GitHub checks passed, so we might want to see if ESLint's `eqeqeq` is enabled.
```suggestion
              ((selectedKey as TabType) === "fixed" ? "fixed" : "action_needed");
```

Reply:
code changed